### PR TITLE
Change docker version when release is tagged

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: scripts/upload_assets.sh [hunk 1]
```diff
@@ -26,13 +26,28 @@ BIN_DIR="./build/_output/bin/bin/"
...
 APP="component-operator"
 
-echo "git tag using snowbot account"
+TEMP_BRANCH="update-operator-yaml-${TAG}"
+OPERATOR_DEPLOYMENT_FILE="./deploy/operator.yaml"
+
 git config user.email "snow-bot@snowdrop.me"
 git config user.name "Snow-bot"
+# revert CircleCI global setting
+git config --remove-section url.ssh://git@github.com 2> /dev/null || true
```
Comment (cmoulliard) [inline scripts/upload_assets.sh]: IS there a reson why we must remove git ssh protocol here to add with set-url  just after https:// ?

Comment (geoand) [reply scripts/upload_assets.sh]: Yes. CircleCI seems to attempt to always use `ssh`. And we leave it in, it will break the `git push` operation

Comment (cmoulliard) [reply scripts/upload_assets.sh]: Could we also include the following change please @geoand ?

- Duplicate existing CSV file under `deploy/olm-catatog/bundle/halkyon.v"OLD_VERSION".clusterserviceversion.yaml` to `deploy/olm-catatog/bundle/halkyon.v"NEW_VERSION".clusterserviceversion.yaml` 
- Replace within the file `deploy/olm-catatog/bundle/halkyon.v"NEW_VERSION".clusterserviceversion.yaml`, the following parameters:
  - `annotations.containerImage: quay.io/halkyonio/operator:latest` -> `annotations.containerImage: quay.io/halkyonio/operator:v"NEW_VERSION"`
  - `name: halkyon.v"OLD_VERSION"` to `name: halkyon.v"NEW_VERSION"`
  - `spec.version: v"OLD_VERSION"` to `spec.version: v"NEW_VERSION"`
  - `spec.instal.....image: quay.io/halkyonio/operator:latest` to `spec.instal.....image: quay.io/halkyonio/operator:v"NEW_VERSION"` 
- Rename within the file `deploy/olm-catatog/bundle/halkyon.package.yaml`, the currentCSV value from `halkyon.v"OLD_VERSION"` to `halkyon.v"NEW_VERSION"`

REMARK: If we continue to release using as convention [SemVer](https://semver.org/) with the prefix `v` -> `v0.1.x`, then I propose that we use also this convention to tag the docker image, for the OLM Bundle files (csv, package, ....). Make sense @geoand @metacosm

Comment (geoand) [reply scripts/upload_assets.sh]: I don't remember seeing docker versions with a `v` prefix...

Comment (cmoulliard) [reply scripts/upload_assets.sh]: origin images contain a v as first letter -> https://quay.io/repository/openshift/origin-pod?tab=tags

Comment (geoand) [reply scripts/upload_assets.sh]: OK, I don't matter for me either way

Comment (cmoulliard) [reply scripts/upload_assets.sh]: We could also adopt as convention to just use `v0.0.0` for the operator release and `0.0.0` for docker tag, CSV and package files. WDYT ? @geoand @metacosm

Comment (geoand) [reply scripts/upload_assets.sh]: Not sure I understand, what is `v0.0.0` and where would it be used?

Comment (geoand) [reply scripts/upload_assets.sh]: Never mind, I re-read the entire description and I see what you mean. Sure, let's go with `v0.0.0`

Comment (geoand) [reply scripts/upload_assets.sh]: Implemented

**Comment by cmoulliard** [comment]
LGTM. Can be merged

**Pull request merged**